packages/site/src/components/ComponentDoc.js

```diff
@@ -48,7 +48,6 @@ type Props = {
 const styles = {
   componentDoc: ({ theme }) => ({
     borderBottom: `2px solid ${theme.borderColor}`,
-    fontFamily: theme.fontFamily_system,
```

Comment:
This is necessary: https://github.com/mineral-ui/mineral-ui/commit/1a56de06967dc56577899161fade1bbdee0f270d.